boards/litex_arty/README.md

```diff
@@ -0,0 +1,137 @@
+LiteX SoC on the Digilent Arty-A7 FPGA Board
+============================================
+
+This board is targeting a SoC bitstream built using
+[LiteX](https://github.com/enjoy-digital/litex), for the [Digilent
+Arty-A7 FPGA
+board](https://reference.digilentinc.com/reference/programmable-logic/arty-a7/start).
+
+Since LiteX is a SoC builder, the individual generated bitstreams can
+differ significantly depending on the LiteX release and configuration
+options used. This board definition currently targets and has been
+tested with
+- [the LiteX SoC generator, revision
+  444a605dea](https://github.com/enjoy-digital/litex/tree/444a605deae6a561dbe2c49bf3062eae6f3cd887)
+- using the included [target
+  file](https://github.com/enjoy-digital/litex/blob/444a605deae6a561dbe2c49bf3062eae6f3cd887/litex/boards/targets/arty.py)
+- built around a VexRiscv-CPU
+- along with the folling configuration options:
+
+  ```
+  --uart-baudrate=1000000
+  --cpu-variant=full
+  --csr-data-width=32
+  --timer-uptime
+  --integrated-rom-size=0xb000
+  --with-ethernet
+  ```
+
+Many bitstream customizations can be represented in the Tock board by
+simply changing the variables in `src/litex_generated.rs`. To support
+a different set of FPGA cores and perform further modifications, the
+`src/main.rs` file will have to be modified.
+
+
+Warning
+-------
```

Comment:
I'm not sure if we really need a warning, pretty much every board supported by Tock is under development.

Reply:
I agree, most boards are in development. I may remove the implementation state under the warning or move it somewhere else. I do however think that lacking memory protection is an issue which should be prominently stated somewhere. I'm hoping that the PMP is integrated into VexRiscv sooner than later, however it could take a few months.

Reply:
It's worth saying that PMP isn't there, just maybe "Warning" is too strong, maybe "Notice" or something instead.

It's not really a issue though, so I'll resolve this.

Reply:
Alright, I see. I've changed it to say "Please note", which is not as drastic as "Warning".